### PR TITLE
add eamxx tests to e3sm suites

### PR DESCRIPTION
**Pull request by mahf708**
Adds EAMxx tests to e3sm_atm_developer, e3sm_atm_integration, and  e3sm_atm_prod.

### CODE REVIEW

File: cime_config/tests.py [hunk 1]
```diff
@@ -238,6 +240,7 @@
         "tests" : (
             "SMS_Ln5.ne30pg2_r05_IcoswISC30E3r5.F2010.eam-wcprod_F2010",
             "SMS_Ld1.ne30pg2_r05_IcoswISC30E3r5.F20TR.eam-wcprod_F20TR",
+            "SMS_Ld1.ne30pg2_ne30pg2.F2010-SCREAMv1.eamxx-prod"
```
[the remark this answers is not in the document]
Comment (rljacob) [reply cime_config/tests.py:243]: We should test it as long as its used.

File: cime_config/tests.py [hunk 2]
```diff
@@ -219,7 +221,7 @@
             "SMS_D_Ln5.ne30pg2_r05_IcoswISC30E3r5.F2010",
             "ERP_Ld3.ne30pg2_r05_IcoswISC30E3r5.F2010.allactive-pioroot1",
             "SMS_Ly1.ne4pg2_oQU480.F2010",
-	    "SMS_D_Ln5.ne45pg2_ne45pg2.FAQP",
+	        "SMS_D_Ln5.ne45pg2_ne45pg2.FAQP",
```
Comment (jgfouca) [inline cime_config/tests.py:224]: This line looks like it is not indented correctly.

**Review by jgfouca — CHANGES_REQUESTED** [review]

**Review by jgfouca — APPROVED** [review]

**Comment by rljacob** [comment]
@mahf708 is this ready?

**Comment by rljacob** [comment]
Question:  should we add some/all of the tests that are currently in the various "e3sm_scream" suites to our nightly testing?

**Comment by rljacob** [comment]
Adding more test can wait but I think you should create new suite names in preparation for that.   The "atm" suites are really EAM suites.   It shouldn't be necessary for an EAM developer to make EAMxx tests pass.  (Or should it?).   So maybe you should make new e3sm_eamxx_developer, e3sm_eamxx_integration, and e3sm_eamxx_prod suites, add the tests you've already added to those, and then include them in the corresponding e3sm full suites.

**Comment by rljacob** [comment]
Good point.  If EAMxx depends on code in components/eam then we need an eamxx test.   That transition will take years but we need to build up our eamxx testing soon.   But new suites can also wait for another PR.

**Review by rljacob — APPROVED** [review]

**Comment by rljacob** [comment]
maybe @oksanaguba can help with the HOMME errors on chrysalis.

**Comment by oksanaguba** [comment]
i agree with Andrew that it would be better to wait till repos are merged together.

**Comment by rljacob** [comment]
reverted from next.

**Pull request closed without merging**